src/librustc_target/abi/call/mod.rs

```diff
@@ -493,6 +494,8 @@ pub struct FnAbi<'a, Ty> {
 
     pub c_variadic: bool,
 
+    pub fixed_count: usize,
```

Comment:
Oh, I forgot to bring this up, could you leave a doc comment here explaining that this can only differ from `args.len()` when `c_variadic` is `true`, and it allows telling apart variadic arguments from always-present (i.e. "fixed") ones.